torch/quantization/fx/quantize.py

```diff
@@ -274,6 +275,7 @@ def is_observed(input_arg):
                 env[node.name] = observed_graph.node_copy(node, load_arg)
```

Comment:
looks like this creates a new name, can you move L240 after this line and use:
```
self.activation_post_process_map[env[node.name]] = observer
```

Reply:
similar for the next case

Reply:
sorry, not this one

Reply:
the idea is the above, I'm not sure which line of code to change, please take a look.

Reply:
@jerryzh168 why do we need to move L240 after this? This else clause doesn't insert observers.

Reply:
The actual issue was somewhere else, in convert we explicitly check if module name startswith `activation_post_process`, so had to update that.